yaks/lib/yaks/errors.rb

```diff
@@ -5,4 +5,6 @@ module Yaks
   RuntimeError              = Class.new(Error)
   UnsupportedOperationError = Class.new(Error)
   PrimitivizeError          = Class.new(Error)
+  MapperNotFoundError       = Class.new(Error)
+  NotACollectionError       = Class.new(Error)
```

Comment:
I need to think this over if I like adding all these custom error types. Maybe a `PolicyError` for any derivation rule that failed irrecoverably? Exception classes are part of the public interface, so something to watch for.